R/fwrite.R

```diff
@@ -0,0 +1,62 @@
+fwrite <- function(dt, file.path, append = FALSE, quote = TRUE,
```

Comment:
should definitely use `DT` or `x`, given the `dt` function in the `stats` package.